agent/src/agent/pipeline/config/jython_scripts/create_events.py

```diff
@@ -6,13 +6,27 @@
 finally:
     sdc.importUnlock()
 
+
+def validate_event(event, record):
+    keys = [key for key in event]
+    if len(keys) == 0:
+        return False
+    for required_key in sdc.userParams.get("REQUIRED_FIELDS", []):
+        if required_key not in keys:
+            sdc.error.write(record, "Record Error: {} is required".format(sdc.userParams["REQUIRED_FIELDS"]))
```

Comment:
Maybe it worth show in error which exactly required_key is missing instead of all REQUIRED_FIELDS?